src/app/routes/api/v3/forms/public-forms.verification.routes.ts

```diff
@@ -7,3 +7,14 @@ export const PublicFormsVerificationRouter = Router()
 PublicFormsVerificationRouter.route(
   '/:formId([a-fA-F0-9]{24})/fieldverifications',
 ).post(VerificationController.handleCreateVerificationTransaction)
+
+/**
+ * Route for resetting the verification of a given field
+ * @returns 400 when the transaction has expired
+ * @returns 404 when the transaction could not be found
+ * @returns 404 when the field could not be found
```

Comment:
On success, should the route return 200?

Reply:
It does but I missed out on the docs @__@ good catch!